### DISCUSSION TITLE
Japanese Training data (as used in the model ja_core_news_lg for example)

### DISCUSSION THREAD

**Discussion by xwd** [explosion/spaCy#8262] [Language Support]
Hi spaCyers, 

on the Japanese model page of Sapcy it reports NER accuracies: 

ENTS_P | Named entities (precision) | 0.74
ENTS_R | Named entities (recall) | 0.70
ENTS_F | Named entities (F-score) | 0.72

was wondering what is the data used for training and evaluating this NER model, is this data downloadable anywhere?

Thanks,

**Comment by adrianeboyd** [comment]
Ah, sorry, it's a mistake in the v3 packaging that the sources aren't listed. They are the same sources as for the v2.3 models linked here under "Sources":

https://v2.spacy.io/models/ja

We'll get that fixed for v3.1!
